Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRTemplateBuilder.cs

```diff
@@ -4,7 +4,9 @@
 using UnityEngine.InputSystem.Utilities;
 using System.Text;
 using UnityEngine.InputSystem.Layouts;
+#if UNITY_INPUT_SYSTEM_ENABLE_XR
```

Comment:
Wouldn't it make more sense to have this #if around all the code in Plugins/XR?

Reply:
I'm thinking of just wrapping it around the functionality (function calls and event registration).

I don't like the idea of wrapping it around types, as it can break serialization if you temporarily lose or disable the XR module.